src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -6586,19 +6604,40 @@ def test_halo_for_discontinuous_2():
 
 
 def test_arg_discontinous():
```

Comment:
Since child and argument indices (integer) were also changed, in the end I used dictionary with the discontinuous FS metadata as keys. From what I gathered when googling, monkeypatching is tricky when not applied to strings.